mock/ext-api-dyson/get/callisto-api-commands1.js

```diff
@@ -0,0 +1,85 @@
+/// Callisto API Mock
+/// See:
+/// curl "http://localhost:3000/callisto-api/transactions?address=0x3083a7ec44ca2b038d4be4b0798152f948f0f3d7"
+/// curl "http://localhost:3000/callisto-api/tokens?address=0xc3d5b69f65027ddf48f894e6e90121293a2f6615"
+/// curl "https://{callisto rpc}/transactions?address=0x3083a7ec44ca2b038d4be4b0798152f948f0f3d7"
+/// curl "https://{callisto rpc}/tokens?address=0xc3d5b69f65027ddf48f894e6e90121293a2f6615"
+/// curl "http://localhost:8420/v1/callisto/0x3083a7ec44ca2b038d4be4b0798152f948f0f3d7"
+/// curl "http://localhost:8420/v2/callisto/tokens/0xc3d5b69f65027ddf48f894e6e90121293a2f6615?Authorization=Bearer"
+
+module.exports = {
+    path: '/callisto-api/:command1?',
+    template: function(params, query, body) {
+        //console.log(query);
+        switch (params.command1) {
+            case 'transactions':
+                if (query.address === '0x3083a7ec44ca2b038d4be4b0798152f948f0f3d7') {
+                    return JSON.parse(`
+                        {
```

Comment:
did you consider storing these JSON responses in files?

Reply:
+
let it be separete PR, @catenocrypt what do you think?

Reply:
Discussed separately, will handle separately.